Assets/VRTK/Prefabs/SnapDropZone/VRTK_SnapDropZone.cs

```diff
@@ -63,10 +64,16 @@ public enum SnapTypes
         [Tooltip("The Snap Type to apply when a valid interactable object is dropped within the snap zone.")]
         public SnapTypes snapType = SnapTypes.UseKinematic;
         [Tooltip("The amount of time it takes for the object being snapped to move into the new snapped position, rotation and scale.")]
+        [HideInInspector]
```

Comment:
Don't use `[HideInInspector]` for obsolete public attributes, use the new custom `[ObsoleteAttribute]` tag as this will still show the parameter in the inspector but change the label appearance to identify it as being deprecated.

Also delete the `[Tooltip()]` associated with the obsolete parameter, then the `[ObsoleteAttribute]` will use the `[Obsolete()]` message for the tooltip, which is more appropriate.